Cubical/Foundations/HLevels.agda

```diff
@@ -25,7 +25,6 @@ hProp {ℓ} = Σ (Set ℓ) isProp
 
 isOfHLevel : ∀ {ℓ} → ℕ → Set ℓ → Set ℓ
 isOfHLevel 0 A = isContr A
-isOfHLevel 1 A = isProp A
```

Comment:
As discussed in some other thread I'm not sure if we should do this change or not... 

@3abc : what do you think?

Reply:
> As discussed in some other thread I'm not sure if we should do this change or not...
> 
> @3abc : what do you think?

How about just keeping two `isOfHLevel`s, one is defined using three cases and one is defined using two? And prove their equivalence. The problem of the current version is not only we will need 

```Agda
isProp→IsOfHLevel1 : ∀ {ℓ} {A : Set ℓ} → isProp A → isOfHLevel 1 A
isOfHLevel1→isProp : ∀ {ℓ} {A : Set ℓ} → isOfHLevel 1 A → isProp A
isSet→isOfHLevel2 : ∀ {ℓ} {A : Set ℓ} → isSet A → isOfHLevel 2 A
isOfHLevel2→isSet : ∀ {ℓ} {A : Set ℓ} → isOfHLevel 2 A → isSet A
```

We will also need things like `isGroupoid→isOfHLevel3` and `isTwoGroupoid→isOfHLevel4` when working with higher truncations. For truncations definitional equality is good because then `isOfHLevel 4` can be used as higher constructors.

Reply:
And if you really want truncation eliminators to compute fast, too much going back and forth will be bad. (This is ironical, what's good about HoTT is one can transport back and forth, in theory it's doable, in practice some of us try to avoid transporting too much so that things really compute.)